src/tribol/interface/tribol.cpp

```diff
@@ -947,14 +947,19 @@ void setInterfacePairs( integer couplingSchemeIndex,
       // since a user may specify interface pairs that aren't, nor should be in contact.
       InterfacePair pair { meshId1[i], pairType1[i], pairIndex1[i],
                            meshId2[i], pairType2[i], pairIndex2[i],
-                           false, i };
+                           true, i };
```

Comment:
Why is this changed from false to true?  Should we use a constructor that doesn't set this if this is just a placeholder value?